public/style.css

```diff
@@ -13,6 +13,10 @@ li {
 }
 
 #user {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
   font-family: 'Barrio', cursive;
   color: white;
   display: flex;
```

Comment:
Remove all these rules and add bootstrap classes for them.

Reply:
Okay, I'll check it.

Reply:
Done.